content/en/platform/corda/4.2/enterprise/cryptoservice-configuration.md

```diff
@@ -233,7 +233,7 @@ clientId: "a3d72387-egfa-4bc2-9cba-b0b27c63540e"
 protection: "HARDWARE"
 ```
 
-The Azure key vault client jar needs to be placed in the drivers directory.  The dependent jars are azure-keyvault and adal4j.
```

Comment:
I think "driver's" is not going to be more clear, because this is a directory called `drivers` and it contains the drivers. Looking at other docs, I see we tend to just add the ticks, so that `drivers` is rendered as code.